rs/https_outcalls/adapter/src/rpc_server.rs

```diff
@@ -204,7 +209,7 @@ impl HttpsOutcallsService for CanisterHttp {
             *http_req.headers_mut() = headers;
             *http_req.method_mut() = method;
             *http_req.uri_mut() = uri.clone();
-            self.client.request(http_req).await.map_err(|e| format!("Failed to directly connect: {e}"))
+            self.client.request(http_req).await.map_err(|e| format!("Failed to directly connect: {:?}", e))
```

Comment:
@rumenov The error messages formatted with display are quite bad. Are you fine with using debug instead so we get proper error messages both for developers and in test output when we have regressions?

Reply:
yes